### PR TITLE
feat(connector): [HELCIM] Add template code for Helcim

### PR DESCRIPTION
**Pull request by deepanshu-iiitu**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Helcim is a low cost connector from North America.

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Motivation and Context

Helcim is a low cost connector from North America.

## How did you test it?

Template code added hence no testing required

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: config/config.example.toml [hunk 1]
```diff
@@ -171,6 +171,7 @@ fiserv.base_url = "https://cert.api.fiservapps.com/"
 forte.base_url = "https://sandbox.forte.net/api/v3"
 globalpay.base_url = "https://apis.sandbox.globalpay.com/ucp/"
 globepay.base_url = "https://pay.globepay.co/"
+helcim.base_url = "https://api.helcim.com/v2/"
```
Comment (prasunna09) [inline config/config.example.toml:174]: Can we just have base url with version "`v2`" not included?

File: config/development.toml [hunk 2]
```diff
@@ -142,6 +143,7 @@ fiserv.base_url = "https://cert.api.fiservapps.com/"
 forte.base_url = "https://sandbox.forte.net/api/v3"
 globalpay.base_url = "https://apis.sandbox.globalpay.com/ucp/"
 globepay.base_url = "https://pay.globepay.co/"
+helcim.base_url = "https://api.helcim.com/v2/"
```
Comment (prasunna09) [inline config/development.toml:146]: same here

**Review by prasunna09 — CHANGES_REQUESTED** [review]

File: config/config.example.toml (hunk 1 above)
Comment (deepanshu-iiitu) [reply config/config.example.toml:174]: Done

File: config/docker_compose.toml [hunk 3]
```diff
@@ -94,6 +94,7 @@ fiserv.base_url = "https://cert.api.fiservapps.com/"
 forte.base_url = "https://sandbox.forte.net/api/v3"
 globalpay.base_url = "https://apis.sandbox.globalpay.com/ucp/"
 globepay.base_url = "https://pay.globepay.co/"
+helcim.base_url = "https://api.helcim.com/v2/"
```
[the remark this answers is not in the document]
Comment (deepanshu-iiitu) [reply config/docker_compose.toml:97]: Done

File: loadtest/config/development.toml [hunk 4]
```diff
@@ -80,6 +80,7 @@ fiserv.base_url = "https://cert.api.fiservapps.com/"
 forte.base_url = "https://sandbox.forte.net/api/v3"
 globalpay.base_url = "https://apis.sandbox.globalpay.com/ucp/"
 globepay.base_url = "https://pay.globepay.co/"
+helcim.base_url = "https://api.helcim.com/v2/"
```
[the remark this answers is not in the document]
Comment (deepanshu-iiitu) [reply loadtest/config/development.toml:83]: Done

File: crates/router/tests/connectors/sample_auth.toml [hunk 5]
```diff
@@ -171,3 +171,7 @@ key1 = "transaction key"
 
 [square]
 api_key="API Key"
+
+
```
[the remark this answers is not in the document]
Comment (deepanshu-iiitu) [reply crates/router/tests/connectors/sample_auth.toml:175]: Done

File: config/development.toml (hunk 2 above)
Comment (deepanshu-iiitu) [reply config/development.toml:146]: Done

**Review by ArjunKarthik — APPROVED** [review]

**Review by prasunna09 — APPROVED** [review]

**Review by lsampras — APPROVED** [review]

**Pull request merged**